src/qt/clientmodel.h

```diff
@@ -13,6 +13,9 @@
 #include <sync.h>
 #include <uint256.h>
 
+#include <net.h>
```

Comment:
Could remove the need for adding this dependency here: 
```diff
diff --git a/src/qt/clientmodel.h b/src/qt/clientmodel.h
--- a/src/qt/clientmodel.h	(revision ea576b65592c23895f80c9116f881ff063253ff8)
+++ b/src/qt/clientmodel.h	(date 1722363555212)
@@ -13,7 +13,6 @@
 #include <sync.h>
 #include <uint256.h>
 
-#include <net.h>
 #include <netaddress.h>
 
 class BanTableModel;
@@ -22,6 +21,7 @@
 class PeerTableModel;
 class PeerTableSortProxy;
 enum class SynchronizationState;
+struct LocalServiceInfo;
 
 namespace interfaces {
 class Handler;
```

This will avoid including the backend `net.h` file in all the gui files that are already including `clientmodel.h`.

Ideally, `net.h` structs/classes shouldn't be accesible by the GUI widgets.  But no prob, it seems we are already including it in `rpcconsole.h` too.

Reply:
Thanks @furszy , this looks better. Applied it and pushed the new version with not `net.h` and with the `struct` only; as you suggested.

Reply:
You changed the net.h stuff in the second commit, after adding it in the first

Reply:
thanks! rebased.